atoma-state/src/migrations/20250206184237_node_public_key_rotation.sql

```diff
@@ -0,0 +1,8 @@
+-- Create tasks table
+CREATE TABLE IF NOT EXISTS node_public_key_rotations (
```

Comment:
I suggest instead of re-creating the `node_public_key_rotations` table, that was already created in a previous migrations, to instead alter it (in place). 

Notice that you are creating the table `if not exists`, which already does, so your changes are not applied and you get a bunch of missing column related errors in the tests. 

Instead, I would change the code to something along the lines:

```rs
ALTER TABLE node_public_key_rotations
    DROP COLUMN tdx_quote_bytes,
    ADD COLUMN tee_quote_bytes BYTEA NOT NULL,
    ADD COLUMN tee_provider BYTEA NOT NULL;
```